app/(dashboard)/enroll/[repositoryId]/page.tsx

```diff
@@ -1,90 +1,39 @@
+"use client";
```

Comment:
I would recommend keeping pages as server components and only load client side functionality using client components as childs